### PR TITLE
Use CSS to set draggable mirror offset

### LINKED ISSUE #2053: Clicking and dragging playlist items in a schedule on mobile doesn't work
iPhone / iOS 14 / Safari 

1. Open a schedule on mobile
2. Add multiple playlist items 
3. Attempt to click and drag

Video: https://drive.google.com/file/d/1PBxaBdtvqeofPKmScCKviHIr5RL3CvtO/view?usp=sharing

The icons for copy and trash are also misaligned. @jasoncaldwell how should these icons be aligned on mobile?

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Due to a limitation of the Draggable library, scrolling offset was not taken into account when dragging on a touch device, causing the mirror to be positioned incorrectly.
Instead of using cursorOffsetY, we are now using CSS top property to position the mirror element.

## Motivation and Context
Fix #2053.

## How Has This Been Tested?
Locally and validated on a real Android/Chrome device. [stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu Do you have a physical  iOS device to validate this fix? 
I tried to use BrowserStack but it seems the touch event is not working 100% there. I tested with [an example](https://shopify.github.io/draggable/examples/simple-list.html) from the library and it didn't work as well, so I think it might be something with BrowserStack. Thanks!

File: test/e2e/schedules/cases/add-distribution.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ var AddDistributionScenarios = function() {
 
         it('should show a table for listing displays', function () {
           helper.waitDisappear(distributionModalPage.getDistributionListLoader()).then(function () {
+            helper.wait(distributionModalPage.getDistributionListTable(), 'Distribution Table');
```
Comment (alex-deaconu) [inline test/e2e/schedules/cases/add-distribution.js:67]: @ezequielc thoughts on moving this line up before waiting for the spinner to show. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc tested on my iphone, Chrome and Safari, both Template Editor and Schedule page. Looks good. Thanks for fixing!

**Review by alex-deaconu — APPROVED** [review]

File: test/e2e/schedules/cases/add-distribution.js (hunk 1 above)
Comment (ezequielc) [reply test/e2e/schedules/cases/add-distribution.js:67]: @alex-deaconu We only show that list if there are displays: https://github.com/Rise-Vision/rise-vision-apps/blob/f7282bf4ac9d804d4e2287fc628d9b34375611a5/web/partials/components/distribution-selector/distribution-list.html#L17, so I don't think it is present before the spinner ends. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu Thanks for testing this 👍

File: test/e2e/schedules/cases/add-distribution.js (hunk 1 above)
Comment (alex-deaconu) [reply test/e2e/schedules/cases/add-distribution.js:67]: @ezequielc sounds good, thanks.

**Pull request merged**